Formula/rclone.rb

```diff
@@ -0,0 +1,17 @@
+class Rclone < Formula
+  desc "Sync files to and from Google Drive, Amazon Cloud Drive, etc."
+  homepage "http://rclone.org"
+  url "https://github.com/ncw/rclone/releases/download/v1.29/rclone-v1.29-osx-amd64.zip"
+  version "1.29"
+  sha256 "64c95564ae69f5238000c6adb2f4d178923176ecfd9d64cddfd80bfa538e1c15"
+  head "https://github.com/ncw/rclone.git"
```

Comment:
Have you tried `brew install rclone --HEAD`? Adding this doesn't make sense unless you have tested it and performed the necessary adjustments to the formula to make it work.